### PR TITLE
Issue 27 - Add normalization functions

### PR DESCRIPTION
**Pull request by johanna0321**
### Description of your changes
Add linear drift and compound centering normalization functions, update documentation, and add functionality to median polish normalization function and output

### Issue ticket number and link
#27 

### Type of change
- [ ] Bug fix
- [X] New feature
   - [ ] Backwards Incompatible?
- [ ] Refactoring / code clean-up
- [X] Documentation add / update
- [ ] Automated Test
- [ ] Other (please specify)

### (If applicable) How has this been tested?
Function work in my internal QQQ normalization pipeline

### CODE REVIEW

File: R/mutate_mzroll_list.R [hunk 1]
```diff
@@ -138,10 +138,10 @@ impute_missing_peaks <- function(mzroll_list,
 #'If \code{fill_values} is a data frame, this function calls \code{impute_missing_peaks()}.
 #'If it is a numeric vector, 'this function calls \code{floor_peaks()}. Other types are not currently supported
 #'
-#'@param mzroll_list: data in triple omic structure
-#'@param fill_values: either a numeric constant or a tibble that maps groupId to log2 feature-level imputation values
-#'@param quant_var: column to use for peak values
-#'@param imputation_sd: standard deviation of Gaussian distribution to use for missing peak imputation
+#'@param mzroll_list data in triple omic structure
```
Comment (PMSeitzer) [inline R/mutate_mzroll_list.R:141]: Good to know that roxygen doesn't want these colons!

File: R/mutate_mzroll_list.R [hunk 2]
```diff
@@ -345,7 +357,27 @@ normalize_peaks_median_polish <- function(mzroll_list,
...
+      # If groupId filter values not in design,
+      # perform median polish on all compounds
+      warning("groupId filter values not found in design; performing median polish on all groupIds")
+      sample_scaling_factors <- normalization_peaks
+    } else {
+      
+      # If they are present, filter on these compounds
+      sample_scaling_factors <- normalization_peaks %>%
+        dplyr::filter(groupId %in% filter_values)
+    }
```
Comment (PMSeitzer) [inline R/mutate_mzroll_list.R:372]: Alternatively, you could

File: R/mutate_mzroll_list.R [hunk 3]
```diff
@@ -399,14 +431,88 @@ normalize_peaks_median_polish <- function(mzroll_list,
...
+  mzroll_list <- romic::update_tomic(mzroll_list, updated_samples)
+  
+  return(mzroll_list)
+  
+}
+
+
+#' Predict Dilutions from Median Polish Scaling Factor
+#' 
+#' Using `median_polish_scaling_factor` output from `normalize_peaks_median_polish`,
```
Comment (PMSeitzer) [inline R/mutate_mzroll_list.R:453]: Consider adding a `@description` tag here

**Review by PMSeitzer — APPROVED** [review]
LGTM

**Review by delfarahalireza — APPROVED** [review]
lgtm!

**Pull request merged**